retrieval_service/alloydb.tests.cloudbuild.yaml

```diff
@@ -74,7 +74,7 @@ steps:
         # Set env var expected by tests
         export DB_USER=$$PGUSER
         export DB_PASS=$$PGPASSWORD
-        python -m pytest
+        python -m pytest -m "not cloudsql"
```

Comment:
Long term it may be more clear to specify the exact tests like `pytest datastore/providers/postgres_test.py`. We don't have to run the other mocked tests during this integration tests but we will have to set up another config to run the mocked tests.